internal/db/schema/migrations/oss/postgres/29/02_kms_version.up.sql

```diff
@@ -0,0 +1,38 @@
+begin;
+
+-- make the required schema changes to adopt
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+-- this migration is from:
+-- https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/02_version.up.sql 
+
+create table kms_schema_version(
```

Comment:
Can you add a `comment on table` for this?

Reply:
ty!